transcom/mymove
# B 21569 int

# B-21569 INT
[B-21569](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1024477)
## Summary
Add service items for Unaccompanied Baggage. They are visible after the shipment is Approved and Submitted. They cane be Rejected and re-Approved.

The feature mentions that the Fuel Surcharge is only for when either the Origin or Destination is CONUS.
I updated the preview page also.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run with feature flags on: FEATURE_FLAG_UNACCOMPANIED_BAGGAGE=true FEATURE_FLAG_ENABLE_ALASKA=true make server_run
2. Login as a Customer and create an OCONUS-to-CONUS UB shipment
3. And create an OCONUS-to-OCONUS UB shipment
4. Log in as a Counselor and complete and approve both shipments
5. Log in as a TOO and submit the OCONUS-to-CONUS shipment
6. See the 4 service items
7. Submit the OCONUS-to-OCONUS shipment
8. See only 3 service items (no Fuel Surcharge)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
### CONUS-OCONUS with POE FSC
![Screenshot 2025-01-07 at 1 13 55 PM](https://github.com/user-attachments/assets/e29df5d8-b43b-41b4-974b-f91fe96073e0)
![Screenshot 2025-01-07 at 1 14 22 PM](https://github.com/user-attachments/assets/f8191295-f234-4674-8363-f8c9a44105de)
### OCONUS-CONUS with POD FSC
![Screenshot 2025-01-07 at 1 15 05 PM](https://github.com/user-attachments/assets/1f56d97d-6205-4ff8-af76-02535d81e854)
![Screenshot 2025-01-07 at 1 15 35 PM](https://github.com/user-attachments/assets/df54aed7-3ca0-42e6-a900-2eaa9163331f)
### OCONUS-OCONUS (no FSC)![Screenshot 2025-01-07 at 1 16 04 PM](https://github.com/user-attachments/assets/909d2ec3-764a-46fd-9689-7725797e2b44)![Screenshot 2025-01-07 at 1 16 22 PM](https://github.com/user-attachments/assets/2c3e830a-b4bf-43a1-a0c2-31469de1ea1d)
